### DISCUSSION TITLE
Zigbee questions

### DISCUSSION THREAD

**Discussion by JGeeforce** [pvvx/ATC_MiThermometer#540] [Q&A]
Hi I flashed my LYWSD03MMC to Zigbee, but I have some questions:

1. I successfully paired it with HA, but can I change the settings, like the BLE firmware? Of what are the settings? I would like to know how often the values are uploaded and that kind of things. Remove smiley, etc etc.
2. How can I go back to Bluetooth firmware?
3. What is the range for Zigbee for this device? 
4. What about battery life with Zigbee?

**Comment by pvvx** [comment]
1. Support for Zigbee firmware is in the [ZigbeeTLc](https://github.com/pvvx/ZigbeeTLc) repository.
2. Use Zigbee OTA: [README.md](https://github.com/pvvx/ATC_MiThermometer/tree/master/zigbee_ota#readme)
3. Depends on the type of device and receiver. Thermometers have different antennas, receivers too. The reception sensitivity of Zigbee coordinators and routers varies greatly...
4. Depends heavily on settings and reports. Depends heavily on settings and reports. Also depends on the operation of the Zigbee coordinator/router. The difference can be more than 10 times.

Only 10 months have passed since the first versions of firmware with ZigBee appeared.
https://github.com/pvvx/ZigbeeTLc/issues/37

The cheapest AAA batteries in Tuya TS0201 with [BLE+Zigbee firmware](https://github.com/pvvx/BZdevice):
![image](https://github.com/user-attachments/assets/755c03a8-de69-4616-ae8e-c606ba7ab4a1)
![image](https://github.com/user-attachments/assets/fe95ce3c-832e-420b-86dd-4805b802ab92)

Measurement every 10 seconds, transmission via BLE every 5 seconds. In Zigbee - according to the [settings described here](https://github.com/pvvx/ZigbeeTLc/issues/6).
The difference in discreteness of Ble and Zigbee transmission from one thermometer:
![image](https://github.com/user-attachments/assets/b393642a-de13-48a3-a62d-d00aa01cd8a8)
Zigbee is always worse.

**Comment by JGeeforce** [comment]
Thanks for your replies. For item 2:

I spent several hours to use HA latest version to rollback from zigbee to BT, but without success. I created the folder, placed firmware (.zigbee) files in it, restarted HA. And tried to trigger the update, but then the firmware version goes from a known version to unknown and nothing happens. 

Anyone managed to go back to BT and has a step by step instruction?
